File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let large = [];
    let small = [];
    for(let i = 0; i < sizes.length; i++){
        sizes[i].sort((a,b) => a-b);
        large.push(Math.floor(sizes[i][1]));
        small.push(Math.floor(sizes[i][0]));
    }
    let l = Math.max(...large);
    let s = Math.max(...small);
    return l*s;
    
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    x, y = 0, 0
    for a,b in sizes:
        candidate = sorted([a,b])
        if x < candidate[0]:
            x = candidate[0]
        if y < candidate[1]:
            y = candidate[1]
        print(candidate, x,y)
    return x*y
